pywbemcli/_common.py

```diff
@@ -282,12 +282,14 @@ def create_cimvalue(cim_type, value_str, is_array):
     CIMValue or list of CIMValue elements.
 
     Parameters:
-      CIM_Type: TODO
-        CIMType for this value.
+      CIM_Type: (:term: `string`)
+        CIMType for this value. The CIM data type name for the CIM object.
+        See :ref:`CIM data types` for valid type names.
 
       Value_str (:term: `string`):
-        String defining the input to be parsed.
-        TODO cover escapes
+        String defining the input to be parsed. This may be a quoted string
+        to provide for space characters in the string in which case the
+        quote character used must be excaped if used within the string.
```

Comment:
This description suggests that the surrounding quotes are present in the string that is passed as a value. Is that really intended?

Reply:
At this point that comment should have not been in the description.  It is a string.